src/coreclr/tools/Common/JitInterface/CorInfoInstructionSet.cs

```diff
@@ -752,6 +752,30 @@ private static InstructionSetFlags ExpandInstructionSetByReverseImplicationHelpe
             return resultflags;
         }
 
+        private static Dictionary<(string, TargetArchitecture), string> AllInstructionSetGroups { get; } = new()
+            {
+                { ("x86-x64",    TargetArchitecture.X64),   "sse2" },
+                { ("x86-x64",    TargetArchitecture.X86),   "sse2" },
+                { ("x86-x64-v2", TargetArchitecture.X64),   "sse4.2 popcnt" },
+                { ("x86-x64-v2", TargetArchitecture.X86),   "sse4.2 popcnt" },
+                { ("x86-x64-v3", TargetArchitecture.X64),   "x86-x64-v2 avx2 bmi bmi2 lznct movbe fma" },
+                { ("x86-x64-v3", TargetArchitecture.X86),   "x86-x64-v2 avx2 bmi bmi2 lznct movbe fma" },
+                { ("skylake",    TargetArchitecture.X64),   "x86-x64-v3 pclmul aes" },
+                { ("skylake",    TargetArchitecture.X86),   "x86-x64-v3 pclmul aes" },
+                { ("armv8-a",    TargetArchitecture.ARM64), "neon" },
+                { ("armv8-1-a",  TargetArchitecture.ARM64), "armv8-a lse crc rdma sha1 sha2 aes" },
```

Comment:
> sha1 sha2 aes

Are these mandatory in armv8-1 ?

Reply:
I copied LLVM behavior (it lists them starting from 8.6) but looks like it's indeed not clear so I removed it from the list, will clarify with the arm folks and add where appropriate later